### ISSUE TITLE
Improve validate tab

### ISSUE THREAD

**Issue by sckott** [FredHutch/shiny-cromwell#142]
Right now we print R list to the output. Do something better than that, maybe:

- Parse the R list and just print the error messages as html elements rather than an R list?
- Use [sprocket](https://github.com/stjude-rust-labs/sprocket) to validate and use that output - not sure if it would be feasible to reuse the CLI output for a browser

**Comment by tefirman** [comment]
Just tried out Sprocket's `check` and `lint` functionalities; not gonna lie, pretty slick and a much nicer display... Just not sure how difficult it would be to incorporate into the codebase. Which repo actually performs this functionality with WOMtools? Is that `proof-api`?

**Comment by sckott** [comment]
it comes from the Cromwell API, not proof-api.

**Comment by sckott** [comment]
yeah investigating is a good label. need to poke around and see what the options are first, then can come back here and get feedback on what to go with
